{'description': {'en': "David's Overlay for Misc Utils"},
 'feed': ['https://cgit.gentoo.org/user/write2David.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/user/write2David.git/',
 'name': 'write2David',
 'owner': [{'email': 'write2david@gmail.com',
            'name': 'David Phillips',
            'type': 'person'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/user/write2David.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/user/write2David.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/user/write2David.git'}],
 'status': 'unofficial'}
$ pmaint sync write2David
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing write2David
*** synced write2David
 * Sync succeeded
$ pmaint regen -t 2 write2David
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use' function invoked by app-editors/komodo-edit-bin-6.1.3 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

Error running an operation:
 Unhandled exception in regen_cache
Exception pkgcore.ebuild.processor.TimeoutError: TimeoutError() in <functools.partial object at 0x7fbd79e36c58> ignored
Traceback (most recent call last):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in sys.exitfunc:
ERROR pkgcore: 
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
pkgcore.ebuild.processor.TimeoutError
 * Cache regen failed with 246
